db/migrate/20230516153521_add_department_to_employees.rb

```diff
@@ -0,0 +1,5 @@
+class AddDepartmentToEmployees < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :employees, :department, foreign_key: true
```

Comment:
try to always use `add_belongs_to`. It makes all job you need to do with `add_reference`

Reply:
`add_belongs_to :employees, :department`

Reply:
what I should do with Department's Manager? Should it be a separate Manager model  or just a column in Department model?

Reply:
Let's skip it for now.

Reply:
and merge